client/src/component/MilestoneCard.js

```diff
@@ -1,63 +1,55 @@
 import "./milestoneCard.css";
 import { CircleChart } from "./CircleChart";
 export const MilestoneCard = ({
-	time,
-	timeVerb,
-	data,
-	display,
+    time,
+    timeVerb,
+    data,
+    display,
```

Comment:
Looks like you have changed tabs to spaces in this file which makes the changes hard to see